src/utilities/viewer.cpp

```diff
@@ -990,7 +1003,28 @@ LRESULT CALLBACK TestViewerWindow::WindowProc(
     case WM_KEYUP:
       gInstance->OnKeyUp(wParam);
       return DefWindowProc(hWnd, uMsg, wParam, lParam);
+    case WM_SIZE:
+      [[fallthrough]];

```

Comment:
Doesn't matter, but not needed when there's no body for the case statement

Reply:
I've read somewhere that empty case would not fallthrough by default anymore, not sure from which version this is the default, figured i'd rather be safe-than-sorry on this.